rpc/src/client.rs

```diff
@@ -32,13 +30,13 @@ use tendermint::Genesis;
 #[async_trait]
 pub trait Client {
     /// `/abci_info`: get information about the ABCI application.
-    async fn abci_info(&self) -> Result<abci_info::AbciInfo> {
+    async fn abci_info(&mut self) -> Result<abci_info::AbciInfo> {
```

Comment:
The `Client` trait now requires self mutability because the `WebSocketClient` needs this to use its internal channel to communicate with its driver. Unfortunately this change becomes viral and affects the `HttpClient` too.

Reply:
This is no longer relevant as of https://github.com/informalsystems/tendermint-rs/pull/646/commits/56ce9f4ed202e663bcd761dbf0772ca4e507d0fd since @romac pointed out that we could use immutable channels for sending data.